README.md

```diff
@@ -33,7 +33,8 @@ export default {
       name: 'server.js',
       nodeArgs: ['--inspect'], // allow debugging
       args: ['scriptArgument1', 'scriptArgument2'], // pass args to script
-      signal: 'SIGUSR2', // signal to send for HMR (defaults to 'SIGUSR2', set to false to disable)
+      signal: false | true | 'SIGUSR2', // signal to send for HMR (defaults to 'SIGUSR2', leave unset to disable)
```

Comment:
The default seems to be `false` now, not `SIGUSR2` as the documentation mentions.

> // signal to send for HMR (defaults to 'SIGUSR2', leave unset to disable)

Reply:
Good catch, thanks!